Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM cusspvz/node:5.1.1
```

Comment:
could you add reasons you think your node docker image is better than the official https://hub.docker.com/_/node/ ?

or if you only want the alpine linux based docker image + node, there seems already many implementations, could you document why you think your own implementation [3] is better than others, just a first look, this cusspvz/node.docker@0e77a04 `dont check nodejs certs` bypassed ssl certificate check concerned me

1. https://github.com/mhart/alpine-node
2. https://github.com/gliderlabs/docker-alpine
3. https://github.com/cusspvz/node.docker

PS: current `docker search node`

```console
➸ docker search node
NAME                       DESCRIPTION                                     STARS     OFFICIAL   AUTOMATED
node                       Node.js is a JavaScript-based platform for...   1959      [OK]       
strongloop/node            StrongLoop, Node.js, and tools.                 27                   [OK]
bitnami/node               Bitnami Node.js Docker Image                    11                   [OK]
siomiz/node-opencv         _/node + node-opencv                            5                    [OK]
dahlb/alpine-node          small node for gitlab ci runner                 4                    [OK]
azukiapp/node              Docker image to run Node.js by Azuki - htt...   4                    [OK]
cusspvz/node               🌐 Super small Node.js container (~20MB)...      3                    [OK]
anigeo/node-forever        Daily build node.js with forever                3                    [OK]
tutum/node                 Run a Tutum node inside a container             3                    [OK]
codenvy/node               Node 0.12.9                                     1                    [OK]
centralping/node           Bare bones NodeJS container.                    1                    [OK]
joxit/node                 Slim node docker with some utils for dev        1                    [OK]
xataz/node                 very light node image                           1                    [OK]
redjack/node               Node + Nave                                     1                    [OK]
tectoro/node-compass       Node JS minimal version with compass and b...   1                    [OK]
seegno/node                A node docker base image.                       1                    [OK]
starefossen/ruby-node      Docker Image with Ruby and Node.js installed    1                    [OK]
centralping/node-onbuild   Bare bones NodeJS onbuild container.            1                    [OK]
codexsystems/node          Node.js for Development and Production          0                    [OK]
lynxtp/node                https://github.com/lynxtp/docker-node           0                    [OK]
prima/node                 node                                            0                    [OK]
stylisted/node             Stylisted's base Docker node image with gr...   0                    [OK]
robbertkl/node             Docker container running Node.js                0                    [OK]
instructure/node           Instructure node images                         0                    [OK]
c4tech/node                NodeJS images, aimed at generated single-p...   0                    [OK]
```